packages/kbn-test/src/es/es_test_config.ts

```diff
@@ -54,10 +59,12 @@ export const esTestConfig = new (class EsTestConfig {
       // or use default values (username and password from ../kbn/users.js)
       protocol: process.env.TEST_ES_PROTOCOL || 'http',
       hostname: process.env.TEST_ES_HOSTNAME || 'localhost',
-      port: parseInt(process.env.TEST_ES_PORT, 10) || 9220,
+      port: process.env.TEST_ES_PORT ? parseInt(process.env.TEST_ES_PORT, 10) : 9220,
```

Comment:
This could potentially set `port: 0`, how about keeping the fallback like:
```suggestion
      port: process.env.TEST_ES_PORT && parseInt(process.env.TEST_ES_PORT, 10) || 9220,
```

Reply:
I added logic to throw an exception on invalid `TEST_ES_PORT` instead of silently fall back to `9220`